pylint/checkers/variables.py

```diff
@@ -717,14 +731,24 @@ class VariablesChecker(BaseChecker):
 
     def __init__(self, linter=None):
         super().__init__(linter)
-        self._to_consume = (
-            None  # list of tuples: (to_consume:dict, consumed:dict, scope_type:str)
-        )
+        self._to_consume: List[NamesConsumer] = []
         self._checking_mod_attr = None
         self._loop_variables = []
         self._type_annotation_names = []
         self._postponed_evaluation_enabled = False
 
+    def open(self) -> None:
+        """Called when loading the checker"""
+        self._undefined_variable_is_enabled = self.linter.is_message_enabled(
+            "undefined-variable"
+        )
+        self._used_before_assignment_is_enabled = self.linter.is_message_enabled(
+            "used-before-assignment"
+        )
+        self._undefined_loop_variable_is_enabled = self.linter.is_message_enabled(
+            "undefined-loop-variable"
+        )
```

Comment:
```suggestion
        self._is_undefined_variable_enabled = self.linter.is_message_enabled(
            "undefined-variable"
        )
        self._is_used_before_assignment_enabled = self.linter.is_message_enabled(
            "used-before-assignment"
        )
        self._is_undefined_loop_variable_enabled = self.linter.is_message_enabled(
            "undefined-loop-variable"
        )
```
This might be a bit better. Especially as those names will inevitable be uses as templates.